dandiapi/api/services/dandiset/__init__.py

```diff
@@ -0,0 +1,60 @@
+from typing import Tuple
+
+from django.core.exceptions import PermissionDenied, ValidationError
+from django.db import transaction
+
+from dandiapi.api.models.dandiset import Dandiset
+from dandiapi.api.models.version import Version
+from dandiapi.api.services.version.metadata import _normalize_version_metadata
+
+
+def create_dandiset(
+    *,
+    user,
+    identifier: int = None,
+    embargo: bool,
+    version_name: str,
+    version_metadata: dict,
+) -> Tuple[Dandiset, Version]:
```

Comment:
```suggestion
) -> tuple[Dandiset, Version]:
```
We can use lowercase collections types since we're on python 3.10.